src/Essentials/samples/Samples/View/BasePage.cs

```diff
@@ -14,20 +13,19 @@ public BasePage()
 			NavigationPage.SetBackButtonTitle(this, "Back");
 			if (DeviceInfo.Idiom == DeviceIdiom.Watch)
 				NavigationPage.SetHasNavigationBar(this, false);
+
+			Loaded += OnLoaded;
+			Unloaded += OnUnloaded;
```

Comment:
OnAppearing and OnDisappearing will fire each time the window loses focus - especially on Windows. This is because technically you can hide the window without actually minimizing it.

The new way is to use Loaded and Unloaded as this is actually when the elements are removed from the UI hierarchy.